templates/var/lib/ansible/playbooks/scaleup.yml

```diff
@@ -16,6 +16,14 @@ cat << 'EOF' > /var/lib/os-apply-config/templates/var/lib/ansible/playbooks/scal
     shell: iptables -A DOCKER -p tcp -j ACCEPT
     when: openshift_use_flannel
 
+  - name: Set up masquerading on flannel interface
+    shell: iptables -t nat -A POSTROUTING -o {{ flannel_interface }} -j MASQUERADE
+    when: openshift_use_flannel
+
+  - name: Make iptables rule permanent
+    shell: /usr/libexec/iptables/iptables.init save
+    when: openshift_use_flannel
+ 
```

Comment:
It's up to you (and other reviewers opinion). IMO doing refactoring now instead of later (and risking it stays forgotten) would be better, but don't take me too seriously - I'm not active on this project.